### ISSUE TITLE
system time tooltip is too verbose

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#8830]
System time tooltip is not very friendly to screen readers:

![screenshot of system time](https://user-images.githubusercontent.com/200109/37402103-99fa7c94-278a-11e8-9bfa-7a184f0e1a6c.png)

**Issue**: The IPv6 address and hostname are a mouthful, even for human speakers. It's problematic at best for screen readers to pronounce every items.

---

(Note: The widget is going to change to a standard tooltip. See #8829. This particular issue will still remain even after the change.)

**Comment by martinpitt** [comment]
I suppose it would be a good improvement to only show the host name if it is available, and only the IP if it is not?

**Comment by garrett** [comment]
@martinpitt: That's better than what we have, but it's also too wordy, especially if it's ipv6.

What if we just notify if there's an error?

- **Success state**: a tooltip with a phrase like "Set via the network"
  - (...would "Network sync" make sense?)
- **Error state**: display to the right of the link "[Warning icon] Could not contact $HOSTNAME"

**Comment by martinpitt** [comment]
This went away with the Overview redesign.
